src/main/scala/org/broadinstitute/dsde/rawls/monitor/BucketDeletionMonitor.scala

```diff
@@ -16,30 +26,27 @@ object BucketDeletionMonitor {
   case class BucketDeleted(bucketName: String) extends BucketDeletionsMessage
 }
 
-class BucketDeletionMonitor(datasource: DataSource, containerDAO: GraphContainerDAO, gcsDAO: GoogleServicesDAO) extends Actor {
+class BucketDeletionMonitor(datasource: SlickDataSource, gcsDAO: GoogleServicesDAO)(implicit executionContext: ExecutionContext) extends Actor with LazyLogging {
 
   override def receive = {
-    case DeleteBucket(bucketName) => deleteBucket(bucketName)
-    case BucketDeleted(bucketName) => recordBucketDeletion(bucketName)
+    case DeleteBucket(bucketName) => deleteBucket(bucketName) pipeTo self
+    case BucketDeleted(bucketName) => recordBucketDeletion(bucketName) pipeTo self
+    
+    case Unit => // successful future
+    case Status.Failure(t) => logger.error("failure in BucketDeletionMonitor", t)
   }
 
-  private def getPendingDeletions(txn: RawlsTransaction) =
-    containerDAO.workspaceDAO.loadPendingBucketDeletions(txn) getOrElse {
-      throw new RawlsException("Cannot find the list of buckets pending deletion")
+  private def deleteBucket(bucketName: String): Future[Unit] = {
```

Comment:
These two have accidentally gotten reversed.  The logic should go:

Delete Bucket = SAVE to PBD table, tell Google to delete

Record Deletion = REMOVE from PDB table